### PR TITLE
Multi-experiment Regression

### PR DESCRIPTION
**Pull request by olynch**
This is a sketch for doing a single regression across multiple experiments, each of which only test a subset of the parameters.

Currently, we are doing it with linear regression. In principle, we would want to replace the linear regression with regression for the rates of different Petri nets.

### CODE REVIEW

**Comment by epatters** [comment]
Since everything in `experiments` is "buyer beware," we can merge this whenever you're ready. Just let me know.

**Comment by olynch** [comment]
Hahaha lol "buyer beware." I'm going to play around a bit more with this before we merge.

File: experiments/MultiExperimentRegression/src/PetriRegression.jl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  prob = remake(prob1,p=p)
+  predicted = solve(prob,Tsit5(),saveat=0.1)
+
+  for i = 1:length(predicted)
+    measurements[:,i] ~ MvNormal(predicted[i], σ)
+  end
+end
+  
+lv = PetriNet(2,(1,(1,1)),((1,2),(2,2)),(2,()))
```
Comment (mehalter) [inline experiments/MultiExperimentRegression/src/PetriRegression.jl:34]: This code might be more readable if you use the supported `=>` notation for defining the transitions

```julia
lv = PetriNet(2, 1=>(1,1), (1,2)=>(2,2), 2=>())
```

**Comment by epatters** [comment]
@olynch, what do you want to do with this? We can merge this experiment if you want.

**Comment by epatters** [comment]
Closing for now. Feel free to reopen if you want.

**Pull request closed without merging**